src/main/java/net/polarizedions/jamesbot/reponders/ResponderDice.java

```diff
@@ -46,13 +47,42 @@ public boolean run(MessageEvent msg) {
         return false;
     }
 
-    private int roll(int size, int number) {
-        int total = 0;
+    private long roll(int size, int number) {
+        if (number > MAX_NUMBER_SMALL_ROLL) {
+            return big_roll(size, number);
+        }
+        
+        long total = 0;
 
         for (int i = 0; i < number; i++) {
             total += RANDOM.nextInt(size) + 1;
         }
 
         return total;
     }
+    
+    private long big_roll(long size, long number) {
+        long roll = (long) (RANDOM.nextGaussian() * standardDeviation(size, number) + mean(size, number));
+        
+        if (roll < number) {
+            return number;
+        }
+        else if (roll > size * number) {
+            return size * number;
+        }
+        
+        return roll;
+    }
+    
+    private double mean(long size, long number) {
+        return (size + 1) * number / 2d;
+    }
+    
+    private double standardDeviation(long size, long number) {
+        return Math.sqrt((sides * size - 1) * number / 12d);
```

Comment:
Updated this on my branch, but too late :( should be size instead of sides here.

Reply:
Fixed in 1b09932